### PR TITLE
Style service details

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: src/components/Phone.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    number(): string {
+        if (this.props) {
+            return this.props.number;
+        }
+        return ""
+    }
+
+    href(): string {
+        return "tel:" + this.number().replace(/[^0-9]/g, '');
```
Comment (danni) [inline src/components/Phone.js]: Add `+` to the list of allowed chars just in case.

File: src/pages/ServicePage.js [hunk 2]
```diff
@@ -26,28 +26,84 @@ class CategoryPage extends React.Component {
...
+                </mui.Paper>
+                <mui.Paper>
+                    <OpeningTimes object={object} />
+                </mui.Paper>
+                <mui.Paper>
+                    <Address {...object.location} />
+                </mui.Paper>
+                {this.renderPhones(object.phones)}
+                <mui.Paper>
+                    <h3>To receive this service you must be</h3>
```
Comment (danni) [inline src/pages/ServicePage.js]: Be aware these may not be set. In which case I don't think we should show that particular heading.

File: src/pages/ServicePage.js [hunk 3]
```diff
@@ -26,28 +26,84 @@ class CategoryPage extends React.Component {
...
                 this.setState({
                     object: data,
                 });
             });
 
-        // FIXME: display error
+        // FIXME: display error on failure to connect
+    }
+
+    renderPhones(phones: Array<Object>): React.Element {
```
Comment (danni) [inline src/pages/ServicePage.js]: We probably want to do something useful with the different phone types. Only show some of the phone types etc.

Phone types are `phone`, `mobile`, `fax`, `tty` and `freecall`.

File: src/components/Address.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        }
+
+        return <span className={property} >{described}</span>;
+    }
+
+    render(): React.Element {
+        return (
+            <div className="Address">
+                {this.describe('level', 'Level')}
+                {' '}
```
Comment (danni) [inline src/components/Address.js]: `&nbsp`? Since we don't want to break across the address anyway.

File: src/components/Address.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+                {this.props.street_number}
+                {' '}
+                {this.props.street_name}
+                {' '}
+                {this.props.street_type}
+                {' '}
+                {this.props.street_suffix}
+                {' '}
+                {this.props.suburb}
```
Comment (danni) [inline src/components/Address.js]: Split into a second line here though.

**Comment by danni** [comment]
Merge in master and write a test for it on top of the mock ISS.

File: src/components/Address.js (hunk 4 above)
Comment (DanielHeath) [inline src/components/Address.js]: multiple consecutive `&nbsp`s will render if there is no content which isn't ideal...

**Comment by danni** [comment]
Is ServicePane still used?

**Comment by danni** [comment]
++

**Pull request merged**